src/utils/create-export-document.js

```diff
@@ -18,9 +18,41 @@ const colors = {
  *   terms
  */
 function createBaseDocument() {
+  const url = window.location.origin + window.location.pathname;
```

Comment:
This excludes the hash route, which wouldn't be helpful when returning to the application.